### PR TITLE
Make 'create user' a bit more robust

### PR DESCRIPTION
**Pull request by bobdenotter**

### CODE REVIEW

**Comment by bobdenotter** [comment]
Pinging @I-Valchev.. I'm merging this in, because i want to prep the release, but tomorrow i'd like to hear your opinion on the weirdness that's going on here.. I don't see how this could ever have worked correctly, and why it chose yesterday to start breaking on this.

File: tests/e2e/users.feature [hunk 1]
```diff
@@ -151,6 +152,7 @@ Feature: Users & Permissions
     And I should see "Suggested secure password"
 
   @javascript
+      @foo
```
Comment (I-Valchev) [inline tests/e2e/users.feature:155]: @bobdenotter Oops... think we have to remove that! 🤪

**Comment by I-Valchev** [comment]
@bobdenotter I don't know why it was breaking? Or what was breaking? 

It's strange that we need the default locale there, because the form is submitted and there the select already has a prepopulated option to begin with.

File: tests/e2e/users.feature (hunk 1 above)
Comment (bobdenotter) [reply tests/e2e/users.feature:155]: Removed it! 🙈

**Pull request merged**